pkg/cosign/kubernetes/webhook/validator.go

```diff
@@ -137,6 +144,27 @@ func (v *Validator) validatePodSpec(ctx context.Context, ps *corev1.PodSpec, opt
 				).ViaFieldIndex(field, i))
 				continue
 			}
+			// Match the pattern
+			// if matched, use the respective key to verify
+			// Read the keys from the CRD from the namepattern that matches
+			keys := []*ecdsa.PublicKey{}
+			matched := false
+			for _, imagePattern := range imagePolicyType.Spec.Verification.Images {
+				imagePatternGlob := glob.MustCompile(imagePattern.NamePattern)
```

Comment:
MustCompile panics when things fail, which is a very undesirable property of webhooks that validate pods and fail closed.  Is the validation checking that this is a valid regular expression to make this safe?